pkg/clusteragent/autoscaling/workload/telemetry.go

```diff
@@ -121,6 +130,15 @@ func trackPodAutoscalerReceivedValues(podAutoscaler model.PodAutoscalerInternal,
 
 	scalingValues := podAutoscaler.MainScalingValues()
 
+	// Track received recommendations version
+	telemetryReceivedRecommendationsVersion.Set(
+		float64(version),
+		podAutoscaler.Namespace(),
+		targetName,
+		podAutoscaler.Name(),
```

Comment:
should this also have `string(scalingValues.Vertical.Source),` ?

Reply:
It should not but it should not be present in the list of tag keys, good catch as the source is per-values.